src/material/chips/chip-row.ts

```diff
@@ -92,6 +94,15 @@ export class MatChipRow extends MatChip implements AfterViewInit {
   /** The projected chip edit input. */
   @ContentChild(MatChipEditInput) contentEditInput?: MatChipEditInput;
 
+  /**
+   * Set on a mousedown when the chip is already focused via mouse or keyboard.
+   *
+   * This allows us to ensure chip is already focused when deciding whether to enter the
+   * edit mode on a subsequent click. Otherwise, the chip appears focused when handling the
+   * first click event.
+   */
+  _alreadyFocused = false;
```

Comment:
Should we make this private? Also maybe something more descriptive like `_wasFocusedOnLastClick`?

Reply:
I'll make it private. I think name is okay though, as the main point is that it has already been focused via a previous click or tab. I think with the comments and only used internally, that works. Else it would have to be really long.